python-sdk/src/astro/settings.py

```diff
@@ -8,13 +8,13 @@
 REDSHIFT_SCHEMA = conf.get("astro_sdk", "redshift_default_schema", fallback=SCHEMA)
 
 
-ALLOW_UNSAFE_DF_STORAGE = conf.getboolean(
-    "astro_sdk", "dataframe_allow_unsafe_storage", fallback=False
+DATAFRAME_STORAGE_CONN_ID = conf.get(
```

Comment:
I'm slightly concerned about us enforcing Python SDK users to have to be on the cloud to run operations which use dataframes. This may not be the case for some of our customers.

What would you recommend for those users?

Reply:
@tatiana there's no safe way to store dataframes that doesn't involve some form of cloud storage. Storing on the local machine won't work unless all of your airflow workers are on that same machine, and storing in the metadata DB will cause performance degradation.